addons/mail/static/src/components/message/message.js

```diff
@@ -414,22 +277,23 @@ export class Message extends Component {
         if (!this.message) {
             return;
         }
-        if (this._prettyBodyRef.el && this.message.prettyBody !== this._lastPrettyBody) {
+        
+        if (this._prettyBodyRef.el) {
             this._prettyBodyRef.el.innerHTML = this.message.prettyBody;
-            this._lastPrettyBody = this.message.prettyBody;
         }
-        // Remove all readmore before if any before reinsert them with _insertReadMoreLess.
-        // This is needed because _insertReadMoreLess is working with direct DOM mutations
-        // which are not sync with Owl.
-        if (this._contentRef.el) {
-            for (const el of [...this._contentRef.el.querySelectorAll(':scope .o_Message_readMoreLess')]) {
-                el.remove();
-            }
-            this._insertReadMoreLess($(this._contentRef.el));
-            this.messaging.messagingBus.trigger('o-component-message-read-more-less-inserted', {
-                message: this.message,
-            });
+        if (this._authorRef.el) {
+            this._authorRef.el.innerHTML = this.message.processDomTransformation(this.message.author.nameOrDisplayName);
+        }
+        if (this._subtypeDescription.el) {
+            this._subtypeDescription.el.innerHTML = this.message.processDomTransformation(this.message.subtype_description);
         }
+        if (this._trackingValuesRef.el) {
+            const trackingValuesDomElements = this._trackingValuesRef.el.querySelectorAll('[data-tracking-value]');
+            for (const element of trackingValuesDomElements) {
+                element.innerHTML = this.message.processDomTransformation(element.dataset.trackingValue);
+            }
+        } 
```

Comment:
All these could be fields to handle the caching. However, need to find a trick for the tracking values. Or just an field with default: {}.

Reply:
done